Slider/Slider.js

```diff
@@ -379,7 +389,20 @@ SliderBase.propTypes = /** @lends sandstone/Slider.SliderBase.prototype */ {
 	 * @type {Number}
 	 * @public
 	 */
-	value: PropTypes.number
+	value: PropTypes.number,
+
+	/**
+	 * The interval (in milliseconds) between valid wheel events.
+	 *
+	 * For example, 200 means to ignore wheel events occurred witnin 200ms
+	 * of the last processed wheel event while 0 means to process all wheel events.
+	 * If the number is large, the slider value changes slowly.
+	 *
+	 * @type {Number}
+	 * @default 0
+	 * @public
+	 */
+	wheelEventInterval: PropTypes.number
```

Comment:
I'm not sure this name is proper or not. it looks good to  me.
Is there anything better? @MikyungKim @0x64

Reply:
Please change the prop to "wheelinterval"

Reply:
It's fixed.